integrationtest/mapper/pojo-base/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/smoke/ExcludePathsIT.java

```diff
@@ -0,0 +1,291 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.mapper.pojo.smoke;
+
+import java.lang.invoke.MethodHandles;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.hibernate.search.integrationtest.mapper.pojo.smoke.bridge.CustomPropertyBinding;
+import org.hibernate.search.integrationtest.mapper.pojo.smoke.bridge.CustomTypeBinding;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.AssociationInverseSide;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.DocumentId;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.GenericField;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexedEmbedded;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.ObjectPath;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.PropertyValue;
+import org.hibernate.search.mapper.pojo.standalone.mapping.SearchMapping;
+import org.hibernate.search.mapper.pojo.standalone.session.SearchSession;
+import org.hibernate.search.util.common.impl.CollectionHelper;
+import org.hibernate.search.util.impl.integrationtest.common.rule.BackendMock;
+import org.hibernate.search.util.impl.integrationtest.mapper.pojo.standalone.StandalonePojoMappingSetupHelper;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+
+public class ExcludePathsIT {
```

Comment:
I'd avoid putting too much in the "smoke" package, better organize tests by feature...

Any reason you didn't just put this next to `org.hibernate.search.integrationtest.mapper.pojo.mapping.definition.IndexedEmbeddedBaseIT#includePaths`, reusing `doTestEmbeddedRuntime` if possible?

Reply:
+100 😄 
I just liked the model in `AnnotationMappingSmokeIT` so I've put it here while I was looking into how things work internally with all these paths. I'll probably just drop this test in the end. Thanks for pointing me to `IndexedEmbeddedBaseIT` 😃